plugins/imagebase/plugin.js

```diff
@@ -614,6 +608,16 @@
 		link: getLinkFeature()
 	};
 
+	function getWidgetsWithFeature( widgets, feature ) {
```

Comment:
Functions should be declared before they are used in the code.

Reply:
I see now that functions in this file are declared at the top and I moved my function also. But are we going with this code style in whole CKEditor codebase? Linter didn't complain about it and I think it's cleaner when your private functions are below the main login. Also I didn't catch this code style in different files. Should I follow this rule in other files?